src/fallbacks.jl

```diff
@@ -95,8 +95,17 @@ issimple(t::AbstractMultiPointTrait, geom) = allunique((getgeom(t, geom)))
 issimple(t::AbstractMultiCurveTrait, geom) = all(issimple.(getgeom(t, geom)))
 isclosed(t::AbstractMultiCurveTrait, geom) = all(isclosed.(getgeom(t, geom)))
 
-crs(::Nothing, geom) = nothing
-crs(::AbstractTrait, geom) = nothing
+crs(::Nothing, geom) = _get_dataapi_metadata(geom, "GEOINTERFACE:crs", nothing)
+crs(::AbstractTrait, geom) = _get_dataapi_metadata(geom, "GEOINTERFACE:crs", nothing)
+
+function _get_dataapi_metadata(geom::GeomType, key, default) where GeomType
+    if DataAPI.metadatasupport(GeomType).read
+        if key in DataAPI.metadatakeys(geom)
+            return DataAPI.metadata(geom, key; style = false)
+        end
+    end
+    return default
+end
```

Comment:
Should I move this to some other location / file?

Reply:
Maybe to (a new) metadata.jl? I discussed having a Geometadata.jl package with @rafaqz, but that's probably overkill for now.